Dizzy/Screens/Search Place/PlaceSearchVM.swift

```diff
@@ -18,18 +20,29 @@ protocol MapSearchVMType {
     func closeButtonPressed()
 }
 
-protocol MapSearchVMDelegate: class {
+protocol PlaceSearchVMDelegate: class {
     func didSelect(place: PlaceInfo)
     func cancelButtonPressed()
 }
 
-final class MapSearchVM: MapSearchVMType {
+final class PlaceSearchVM: PlaceSearchVMType {
```

Comment:
i think you should insert the search code in to `PlacesVM` and use the existing instance of it (make `PlacesVM` singleton in the assembly), this way we will have the search ability on the `discoveryVC` too